web-client/src/presenter/computeds/createOrderHelper.js

```diff
@@ -22,9 +22,10 @@ export const createOrderHelper = (get, applicationContext) => {
   const isLeadCase = caseDetail.leadDocketNumber === caseDetail.docketNumber;
 
   return {
+    addDocketNumbersButtonIcon: addedDocketNumbers ? 'edit' : 'plus-circle',
     addDocketNumbersButtonText: addedDocketNumbers
-      ? 'Edit docket numbers to the coversheet'
-      : 'Add docket numbers to the coversheet',
+      ? 'Edit docket numbers in the cover sheet'
```

Comment:
Nit: 
```suggestion
      ? 'Edit docket numbers on the cover sheet'
```

Reply:
ux asked for `to` and `in` specifically

Reply:
Obviously I'll defer, but `on` would be correct